xbmc/cores/RetroPlayer/rendering/RPRenderManager.cpp

```diff
@@ -661,3 +662,43 @@ CRenderVideoSettings CRPRenderManager::GetEffectiveSettings(
 
   return effectiveSettings;
 }
+
+void CRPRenderManager::SaveThumbnail(const std::string& path)
+{
+  m_bufferMutex.lock();
+
+  if (m_renderBuffers.size() <= 0)
```

Comment:
```suggestion
  if (m_renderBuffers.size() == 0)
```
`size_t` is unsigned